src/main/java/com/google/code/or/OpenReplicator.java

```diff
@@ -91,12 +98,12 @@ public void start() throws Exception {
 		this.transport.connect(this.host, this.port);
 
 		//

```

Comment:
these empty comments are all over the place.  agree that they should go, will do in a separate PR